addons/l10n_at/data/template/account.account-at.csv

```diff
@@ -1,198 +1,235 @@
+"chart_at_template_8180","Carrying amount of disposed other long-term financial assets (+)","8180","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8180","Buchwert abgegangener sonstiger Finanzanlagen (+)"
+"chart_at_template_8181","Carrying amount of disposed other long-term financial assets (-)","8181","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8181","Buchwert abgegangener sonstiger Finanzanlagen (-)"
+"chart_at_template_8190","Carrying amount of disposed long-term securities (+)","8190","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8190","Buchwert abgegangener Wertpapiere des Umlaufvermögens (+)"
+"chart_at_template_8191","Carrying amount of disposed long-term securities (-)","8191","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8191","Buchwert abgegangener Wertpapiere des Umlaufvermögens (-)"
+"chart_at_template_8200","Carrying amount of disposed long-term equity investments (+)","8170","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8170","Buchwert abgegangener Beteiligungen (+)"
+"chart_at_template_8200","Revenue from disposal of long-term equity investments (+)","8200","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8200","Erlöse aus dem Abgang von Beteiligungen (+)"
+"chart_at_template_8205","Revenue from disposal of other long-term financial assets (+)","8205","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8205","Erlöse aus dem Abgang von sonstigen Finanzanlagen (+)"
+"chart_at_template_8206","Revenue from appreciation of other long-term financial assets","8206","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8206","Erlöse aus Zuschreibung sonstige Finanzanlagen"
+"chart_at_template_8210","Revenue from disposal of long-term securities (+)","8210","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8210","Erlöse aus dem Abgang von Wertpapieren des Umlaufvermögens (+)"
+"chart_at_template_8211","Appreciation of other long-term securities","8211","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8211","Zuschreibung Wertpapiere des Umlaufvermögens"
+"chart_at_template_8230","Loss transfer from profit and loss transfer","8230","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8230","Verlustübernahme Ergebnisabführung"
+"chart_at_template_8350","Unused supplier cash discount","8350","False","income_other","account.account_tag_financing,l10n_at.account_tag_external_code_8350","Nicht ausgenützte Lieferantenskonti"
+"chart_at_template_9010","Nominal Capital","9010","False","equity","account.account_tag_financing,l10n_at.account_tag_external_code_9010","Stammkapital"
```

Comment:
"Nominal capital" for chart_at_template_9010 and chart_at_template_9030

Reply:
If I understood it correctly is basically the same thing just for different Austrian company types. So I do not really know how to translate it. Should I just append the German / Austrian name?

Reply:
We could let "Nominal Capital" of 9010 and put an alternative (given by deepl) for 9030, i have those "Capital stock" or 
"Registered capital"

Reply:
Capital stock for 9030 looks good to me. Thanks